src/main/java/uk/gov/justice/digital/hmpps/csr/api/config/ClientTrackingConfiguration.kt

```diff
@@ -0,0 +1,56 @@
+package uk.gov.justice.digital.hmpps.csr.api.config
+
+import com.nimbusds.jwt.JWTClaimsSet
+import com.nimbusds.jwt.SignedJWT
+import io.opentelemetry.api.trace.Span
+import jakarta.servlet.http.HttpServletRequest
+import jakarta.servlet.http.HttpServletResponse
+import org.slf4j.LoggerFactory
+import org.springframework.context.annotation.Configuration
+import org.springframework.http.HttpHeaders
+import org.springframework.web.servlet.HandlerInterceptor
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
+import java.text.ParseException
+
+@Configuration
+class ClientTrackingConfiguration(private val clientTrackingInterceptor: ClientTrackingInterceptor) : WebMvcConfigurer {
```

Comment:
gone for the standard name to keep consistent with other projects instead of `ClientTrackingTelemetryModule.kt`